test/colony-network-auction.js

```diff
@@ -297,6 +298,64 @@ contract("ColonyNetworkAuction", accounts => {
     it("cannot bid with 0 tokens", async () => {
       await checkErrorRevert(tokenAuction.bid(0), "colony-auction-invalid-bid");
     });
+
+    it("auction closes when the receivedTotal goes over the total amount to end the auction", async () => {
+      // Considers totalToEndAuction < receivedTotal as per colonyNetwork#416
+      await giveUserCLNYTokens(colonyNetwork, BIDDER_1, quantity);
+      await clny.approve(tokenAuction.address, quantity, { from: BIDDER_1 });
+
+      let endTime = await tokenAuction.endTime();
+      const amount = new BN(10).pow(new BN(20));
+
+      while (endTime.isZero()) {
+        await forwardTime(SECONDS_PER_DAY * 3, this);
+
+        await tokenAuction.bid(amount, { from: BIDDER_1 });
+        // const price = await tokenAuction.price();
+        // const totalToEndAuction = await tokenAuction.totalToEndAuction();
+        // const receivedTotal = await tokenAuction.receivedTotal();
+        endTime = await tokenAuction.endTime();
+        // console.log("price", price.toString());
+        // console.log("totalToEndAuction", totalToEndAuction.toString());
+        // console.log("receivedTotal", receivedTotal.toString());
+      }
+
+      await tokenAuction.finalize();
+      const receivedTotal = await tokenAuction.receivedTotal();
+      const endPrice = new BN(10)
```

Comment:
Could import `WAD` from constants.

Reply:
I mean, calling it something else is an option, but I don't 100% see the benefit since other codebases speak in WADs and we need some word to describe `10 ** 18`. Mostly I think the semantics are clearer when we share the same constant, since the intention is not `10 ** 18` as a number but rather as an order of magnitude for other numbers. We can call it something different if we feel strongly.